### DISCUSSION TITLE
Add testing email functionality to utilities

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#13664] [Ideas] [answered]
Recently i wanted to test if email works on the server, but craft instance there had admin changes disabled, so i could not access test email functionality.
When i think about it, testing email does not include changing anything, so it should be available from "utilities" section of control panel too.

**Comment by tommysvr** [comment, marked as answer]
Craft 5.6.0 made it possible to send test emails even when `allowAdminChanges` is disabled, as Settings will show in read only mode: https://craftcms.com/docs/5.x/system/control-panel.html#read-only-mode
